usr/src/uts/common/sys/fs/zfs.h

```diff
@@ -140,6 +140,8 @@ typedef enum {
 	ZFS_PROP_STMF_SHAREINFO,	/* not exposed to the user */
 	ZFS_PROP_DEFER_DESTROY,
 	ZFS_PROP_USERREFS,
+	ZFS_PROP_LOGICALQUOTA,
+	ZFS_PROP_LOGICALREFQUOTA,
```

Comment:
For backwards compatility of mismatched user/kernel components (which is not an issue for illumos but applies on linux), we prefer to add new properties at the end of this enum.